cli/pbjs.js

```diff
@@ -99,6 +99,8 @@ exports.main = function main(args, callback) {
                 "                   es6       ES6 wrapper (implies --es6)",
                 "                   closure   Just a closure adding to protobuf.roots (see -r)",
                 "",
+                "  --dependency     Specifies which version of protobuf to require. Accepts any valid module id",
+                "",
```

Comment:
To be more specific. util.wrap uses `protobufjs` as default value when dependency option is not provided.
`static-module` target will use `protobufjs/minimal` as default if not overwritten by options, provided in arguments, but `json-module` is not doing this.

To make things consistent we can either remove default value from `static-module` code and specify default in `pbjs.js` or update `json-module` target